verify/BUILD

```diff
@@ -27,3 +27,14 @@ go_library(
     ],
         importpath = "github.com/project-oak/transparent-release/verify",
 )
+
+go_test(
+    name = "verify_test",
+    size = "small",
```

Comment:
Should we change this to larger size? This tests clones the Oak repo and runs a full build of it, which takes about 9min in CI.

Reply:
I would add that I wrote an implementation this test yesterday using a simple mock build file (uses a 5MB alpine linux docker to produce an output "artifact" with a 5L bash script). Verifies the same thing, but runs a lot faster. Maybe we want to switch to something like that?

Reply:
> I wrote an implementation of this type of test yesterday

I am not sure I follow. What do you mean by "this type of test"?

Reply:
An integration test. In this specific case that means it runs an actual build and verifies the hash. The same thing as this, but it uses a simplified provenance file that executes faster.

Reply:
We can have both too btw, it seems handy to run a test of verifying an actual Oak provenance file (since that is our current customer). But probably we want to separate out that test from the other unit tests, since it takes a long time to run. :)

Reply:
The long running test on the CI should not be a problem. When testing locally, it would be nice to filter them out. Not sure what is the best/easiest way to do this with Bazel.

> The same thing as this, but it uses a provenance file that describes a simplified (or mock build) so it executes a lot faster.

That is a good idea. I think it could be a new test in `common_test.go` with a simple BuildConfing that could potentially use the same docker image from Oak, but run a different/simpler command. Is that what you are thinking?